ci-operator/step-registry/openstack/provision/sriov-operator/openstack-provision-sriov-operator-commands.sh

```diff
@@ -81,7 +100,19 @@ if [ -n "${is_dev_version:-}" ]; then
     unset CLUSTER_TYPE
     make deploy-setup
     popd
+
+    # This is needed to avoid the error where sriov-network-config-daemon fails to start because
+    # it violates PodSecurity.
+    # When the operator is installed from source, it doesn't have the necessary permissions to be used in OpenShift.
+    oc label ns openshift-sriov-network-operator --overwrite \
+        pod-security.kubernetes.io/audit=privileged \
+        pod-security.kubernetes.io/enforce=privileged \
+        pod-security.kubernetes.io/warn=privileged \
+        security.openshift.io/scc.podSecurityLabelSync=false
```

Comment:
I wonder if we could have prepared such namespace in advance instead of waiting for pods to be recreated once we set that?

Reply:
I'll try that. I wasn't sure before because the namespace creation is handled by `make deploy-setup` already. But let's see if I can do it before and set the labels beforehand. Thanks for the suggestion!